README.md

```diff
@@ -54,3 +54,12 @@ This imports the USWDS, pulls in the NJ-specific components and styles, and save
 This builds USWDS styles, builds the Fractal docs, and then deploys them to the gh-pages branch.
 
 Note: do not push directly to the gh-pages branch.
+
+## Releasing a new version to NPM
+
+1. Go to the Draft Release Github Action https://github.com/newjersey/njwds/actions/workflows/draft-release.yml
```

Comment:
Also, it looks like the rest of the readme uses "link text" every time, we should probably change all of the links in this section to match as well, e.g:
```
1. Go to the [Draft Release GitHub Action](https://github.com/newjersey/njwds/actions/workflows/draft-release.yml)
...
```